integration_tests/steps/jira_issues_tests.py

```diff
@@ -302,3 +302,28 @@ def area_external_trade(context):
     ext_trade = list(filter(lambda x: x['areaName'] == "External Trades",
                             cumulative_trade[house1.uuid]))[0]['bars'][0]['energy']
     assert isclose(ext_trade, -1 * 0.666, rel_tol=1e-05)
+
+
+@then('we test the min/max/avg trade and devices bill')
+def check_area_trade_and_bill(context):
+    from integration_tests.steps.integration_tests import get_simulation_raw_results
+    from d3a.d3a_core.sim_results.market_price_energy_day import MarketPriceEnergyDay
+    from d3a.d3a_core.sim_results.bills import CumulativeBills
```

Comment:
Could these be imported at the beginning of the file ? Thanks

Reply:
Yes, `MarketPriceEnergyDay ` and `CumulativeBills ` could be skipped from the lazy imports.